### ISSUE TITLE
API4

### ISSUE THREAD

**Issue by TheFixDev** [EvolSoft/CustomAlerts#41]
Hello can you please Update to API4?

**Comment by Flavius12** [comment]
For updates see: https://flavius12.net/download/customalerts/#updates

**Comment by TheFixDev** [comment]
You have to pay money then you update it?

**Comment by Flavius12** [comment]
Yes. I'm forced to do that since I have to pay IDE software and web domain too. I had got a free .tk domain but the provider unregistered it, so now I'm forced to pay a .net domain

**Comment by TheFixDev** [comment]
Mhm okay :/

**Comment by TheFixDev** [comment]
What can you pay with?

**Comment by Flavius12** [comment]
You can click on donate button on the link I sent you

**Comment by Flavius12** [comment]
Please read carefully what's written there

**Comment by KanekiOnMC** [comment]
Hello, I'm working for the PMMP 4 version of CustomAlerts. Flavius12 is the one accepting my Pull Requests
